Single_Client/client.py

```diff
@@ -1,7 +1,9 @@
 import os
 import socket
 import subprocess
-
+import getpass
+import platform
+import os
```

Comment:
No problem. I'm working in my fork adding get command to download files from client in my files branch, maybe you are interesting in this feature.